test/README.md

```diff
@@ -48,10 +48,10 @@ E2E_TAGS="eventing" test/e2e-tests-local.sh
 
 ### Running a single test case
 
-To run one e2e test case, e.g. TestBasicWorkflow
+To run one e2e test case, e.g. TestTrafficSplit
```

Comment:
Since this seems to be a different topic I suggest adding to a different PR on how to run specific e2e tests. Which is valuable in itself.